includes/uploader.js

```diff
@@ -161,18 +162,27 @@ function checkFileContent() {
 
         // Check if the file content has comma as a separator
         if (content.includes(',')) {
-            columnNames = content.split('\n')[0].split(',');
-            checkColumnNames(columnNames);
-
-            // Extract and log each row
-            rowValues = content.split('\n').slice(1).map(row => row.split(','));
-
-            // Add an ID column in front
-            columnNames.unshift("ID");
-            let rowsWithID = rowValues.map((row, index) => [(index + 1)].concat(row));
-
-            rowValues = rowsWithID;
-            window.rowValues = rowsWithID;
+            let lines = content.split('\n');

```

Comment:
IIUC, this is parsing the uploaded file into rows. Will this break if there are embedded newlines in cell contents? Excel users often have multiline strings when working with text in spreadsheets. Can't we use a standard library for this kind of stuff?

Reply:
Wwe should make a test. I don't know.

Reply:
I agree it's something we should look into, I've opened issue #64 on it.